src/main/java/mate/academy/spring/dao/impl/RoleDaoImpl.java

```diff
@@ -16,10 +16,10 @@ public RoleDaoImpl(SessionFactory factory) {
     }
 
     @Override
-    public Optional<Role> getByName(String roleName) {
+    public Optional<Role> getByName(Role.RoleName roleName) {
         try (Session session = factory.openSession()) {
             return session.createQuery("FROM Role WHERE roleName = :roleName", Role.class)
-                    .setParameter("roleName", Role.RoleName.valueOf(roleName))
+                    .setParameter("roleName", roleName.name())
```

Comment:
Let's try fix by this way 
```
 .setParameter("roleName", roleName)
```
According the common mistakes - "You should pass enum to query not (String)". 
And check if it's work